config/vbox.yaml

```diff
@@ -1,5 +1,6 @@
 # kraken-build.go: describes a build for a Virtualbox testbed
 targets:
+  'u-root':
```

Comment:
We discussed possibly putting an output path here, though that would require generation of a `go.mod` so that a path outside `GOPATH` can be specified. What do you think?

Reply:
So instead of having `os` and `arch`, have, e.g., `destination`. The default value could be `""` which would generate the tree into `build/u-root/`.